src/libs/axe-core.js

```diff
@@ -1,8 +1,7 @@
+import axe from '@axe-core/react';
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-const axe = require('@axe-core/react');
```

Comment:
@cahrens  was this require for axe done inline async on purpose?  Does Nick's import fix break something here?

Reply:
This module (`src/libs/axe-core`) is imported asynchronously from `src/appLoader` based on app configuration. I checked that I was still seeing console messages from axe after making this change.